packages/plugin-kit/src/hooks/slice-update.ts

```diff
@@ -12,12 +14,16 @@ import type {
 export type SliceUpdateHookData = {
 	libraryID: string;
 	model: SharedSliceModel;
+	mocks?: SharedSliceContent[];
 };
 
 /**
  * Return value for `slice:update` hook handlers.
  */
-export type SliceUpdateHookReturnType = void;
+export type SliceUpdateHookReturnType = {
+	mocks: SharedSliceContent[] | undefined;
```

Comment:
Same as above 🤔 If the manager needs to read mocks it better read them using its read mocks function

Reply:
Ok I think I got it, if you need a manager method to return different data, then you can create your own type there, we do that for multiple methods already. These types are primarily meant to type hooks API used by plugins.